### PR TITLE
[Enhancement] Enable compile benchmark codes automatically

### PR DESCRIPTION
**Pull request by LiShuMing**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fix problems below:
- now benchmark codes which depend google-benchmark are not compiled by default, which will cause compile errors if nobody notice
- google-benchmark lib is installed first in dev machine which is not controlled by sr.
- benchmark lib is a very useful tool, so we can add a bench dir to accumulate scripts and base codes for developing later.
- aws-sdk-cpp-1.9.179 build error.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: be/src/bench/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This file is made available under Elastic License 2.0.
```
Comment (imay) [inline be/src/bench/CMakeLists.txt:1]: Should use our license header

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by murphyatwork — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by LiShuMing** [comment]
run starrocks_admit_test

**Comment by LiShuMing** [comment]
run starrocks_admit_test

**Comment by LiShuMing** [comment]
run starrocks_admit_test

**Comment by LiShuMing** [comment]
run starrocks_admit_test image

**Review by murphyatwork — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**